SpecsUtils/src/pt/up/fe/specs/util/collections/SpecsArray.java

```diff
@@ -22,7 +22,7 @@ public class SpecsArray {
      */
     public static int getLength(Object object) {
         var objectClass = object.getClass();
-        if (!objectClass.getClass().isArray()) {
```

Comment:
This line was causing a build error. Basically it was always returning the class of `Class` not the object itself.